src/commands/app/deploy.ts

```diff
@@ -216,25 +239,20 @@ async function run(argv: Arguments) {
         .on('error', reject)
         .on('finish', resolve);
     });
+    */
   } catch (error) {
-    console.error(`Failed deploy cloud code ${name}:`, error);
+    return Promise.reject('Fail to deploy. ' + error);
```

Comment:
Javascript does not provide a way to chain error so we can only write `throw new Error("Fail to deploy. " + error)`